Libraries/LibGUI/Clipboard.h

```diff
@@ -39,13 +41,14 @@ class Clipboard {
     String data() const { return data_and_type().data; }
     String type() const { return data_and_type().type; }
     void set_data(const StringView&, const String& data_type = "text");
-
+    void set_data(const Gfx::Bitmap&);
     struct DataAndType {
         String data;
         String type;
     };
 
     DataAndType data_and_type() const;
+    RefPtr<Gfx::Bitmap> get_bitmap();
```

Comment:
Can this be `const` qualified? Getting a bitmap from the clipboard seems like a non-mutating operation.